Source/WebCore/rendering/GridLayoutFunctions.cpp

```diff
@@ -290,6 +290,16 @@ bool allowedToStretchGridItemAlongRowAxis(const RenderBox& gridItem, ItemPositio
     return justifySelf == ItemPosition::Stretch && hasAutoSizeInRowAxis(gridItem, writingMode) && !hasAutoMarginsInRowAxis(gridItem, writingMode);
 }
 
+bool isRelativeGridTrackBreadthAsAuto(const Style::GridTrackFitContentLength& length, Style::GridTrackSizingDirection, std::optional<LayoutUnit> availableSpace)
```

Comment:
Can these be static functions? 

Why are we taking in the GridTrackSizingDirection if we are not using it?

Reply:
They potentially could be but I went with free functions since that's what the rest of `GridLayoutFunctions` is doing.

Ah! Yeah that was from the move but should have been deleted. Fixed!